src/main/java/fi/aalto/cs/apluscourses/intellij/services/PluginSettings.java

```diff
@@ -164,16 +181,40 @@ public void initializeLocalSettings() {
    * Resets all local settings to their default values.
    */
   public void resetLocalSettings() {
-    unsetLocalSettings();
-    initializeLocalSettings();
+    unsetLocalIdeSettings();
+    initializeLocalIdeSettings();
   }
 
   /**
-   * Unsets all the local settings from {@link LocalSettingsNames}.
+   * Unsets all the local IDE settings from {@link LocalIdeSettingsNames}.
+   */
+  public void unsetLocalIdeSettings() {
+    Arrays.stream(LocalIdeSettingsNames.values())
+        .map(LocalIdeSettingsNames::getName)
+        .forEach(applicationPropertiesManager::unsetValue);
+  }
+
+  /**
+   * Method (getter) to check the property, representing if the project has been "Turned into A+
+   * project".
+   *
+   * @param project a {@link Project} to set the setting to.
+   * @return a boolean value that represent the state of the project.
+   */
+  public boolean isAPlusProjectSetting(@NotNull Project project) {
+    return Boolean.parseBoolean(
+        PropertiesComponent.getInstance(project).getValue(A_PLUS_IS_A_PLUS_PROJECT.getName()));
+  }
```

Comment:
I understand the idea here, but could we just use the `CourseFileManager.courseFileExists` instead?